packages/wdio-crossbrowsertesting-service/tsconfig.json

```
@@ -3,7 +3,8 @@
   "compilerOptions": {
     "baseUrl": ".",
     "outDir": "./build",
-    "rootDir": "./src"
+    "rootDir": "./src",
+    "types": ["webdriverio", "@wdio/logger", "jest", "@wdio/mocha-framework", "@wdio/jasmine-framework", "@wdio/cucumber-framework"]
```

I am not sure if we need them all
```suggestion
    "types": ["webdriverio"]
```

---

We need atleast `@wdio/mocha-framework` and `@wdio/jasmine-framework` to be included because few of the types in `browser config` interface comes from these in the service.ts file

---

I am sure you will be fine without it 😉

---

Problem is for these guys
 core `WebdriverIO.Test` interface doesn't have typings for `fullName` & `title` and same for few the typings for `browser.config.mochaOpts` and that's why I have added those.

Even though typescript is shouting at me in the file, the test cases seems to be passing  without any issues though. Will make this change as you suggested

---

Done